reports/graphs/temperature_change.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from django.utils.translation import gettext as _
+
+import plotly.offline as plotly
+import plotly.graph_objs as go
+
+from reports import utils
+
+
+def temperature_change(objects):
+    """
+    Create a graph showing temperature over time.
+    :param objects: a QuerySet of Temperature instances.
+    :returns: a tuple of the graph's html and javascript.
+    """
+    objects = objects.order_by("-time")
+
+    trace = go.Scatter(
+        name=_("Temperature"),
+        x=list(objects.values_list("time", flat=True)),
+        y=list(objects.values_list("temperature", flat=True)),
+        fill="tozeroy",
```

Comment:
One minor thing here -- lets not use `fill` for this graph in particular. I think the presentation is kind of clearer without it (i.e., it makes it easier to see drastic changes or trends for the temperature).

Reply:
Yeah that makes sense.